docs/versioned_docs/version-zenith/guides/snippets/github-google-cloud/main.yml

```diff
@@ -0,0 +1,22 @@
+name: 'ci'
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  dagger:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Install Dagger CLI
+        run: curl -L https://dl.dagger.io/dagger/install.sh | BIN_DIR=$HOME/.local/bin sh
```

Comment:
We have a `dagger-for-github` action. For example: https://github.com/aluzzardi/daggerverse/blob/a05cc52ae6d60a34a316a649363d833623e432ca/.github/workflows/ci.yml

Reply:
Modified